main/guides/zoe/contract-upgrade.md

```diff
@@ -0,0 +1,168 @@
+# Contract Upgrade
+
+The result of starting a contract includes the right to upgrade the contract. A call to [E(zoe).install(...)](/reference/zoe-api/zoe.md#e-zoe-startinstance-installation-issuerkeywordrecord-terms-privateargs) returns a record of several objects that represent different levels of access.
+The `publicFacet` and `creatorFacet` are defined by the contract.
+The `adminFacet` is defined by Zoe and includes methods to upgrade the contract.
+
+::: tip Upgrade Governance
+
+Governance of the right to upgrade is a complex topic that we cover only briefly here.
+
+- When BLD staker governance starts a contract using `swingset.CoreEval`,
+  to date, the `adminFacet` is stored in the bootstrap vat, allowing
+  the BLD stakers to upgrade such a contract in a later `swingset.CoreEval`.
+- The `adminFacet` reference can be discarded, so that noone can upgrade
+  the contract from within the JavaScript VM. (BLD staker gonvernace
+  could, in theory, change the VM itself.)
+- The `adminFacet` can be shared with a governance contract such as `committee.js`.
```

Comment:
> `committee.js`

a link to the @agoric/governance readme might be nice, or to one of these in the future:
- #749
- #952